### ISSUE TITLE
The Kunena menu is always installed in English, regardless of which language is used

### ISSUE THREAD

**Issue by rich20** [Kunena/Kunena-Forum#8938]
**Describe the bug**
The Kunena menu is always installed in English, regardless of which language is used. Also Restore Kunena Menu does not help.

**System information**
- Joomla version: 4.1.3-dev
- Joomla template: CASSIOPEIA
- Kunena version: 6.0.0-RC3-DEV-GIT
- Kunena template: Aurelia
- Php version: 7.4.28 and 8.0.13 
- Database version: 10.5.11-MariaDB and 10.4.22-MariaDB

**Desktop**
- OS: Ubuntu 20.04.2 LTS
- Browser: Chromium, Firefox and Opera

**Comment by rich20** [comment]
@xillibit has something been changed? This bug is no longer present.

**Comment by xillibit** [comment]
Good to know that is fixed but i don't changed anything on this side

**Comment by rich20** [comment]
I have found the cause. In our **Kunena installation package** is missing the directories for the com_kunena.install.in for all languages. These files must be included in **/src/admin/language** (see picture). I included these backend files in my installation for testing purposes, therefore it works for me.
But not all ini-files are needed for the installation, it is enough if these folders contain only the respective xx-XX.com_kunena.install.ini
This solves the problem with the menu, it is created in the correct language.
<br />
![Bildschirmfoto vom 2022-05-12 04-18-19](https://user-images.githubusercontent.com/888847/167978934-6972c398-88a9-4bca-b949-29f01003aff8.png)
